locations/spiders/creditunion.py

```diff
@@ -107,24 +107,29 @@ def parse_bank(self, bank):
         except IndexError:
             phone = ""
 
-        href = bank.xpath('.//div[@class="location-results__share"]/div/a/@href').extract_first()
-        lat, lon = re.search(r"lat=(.*?)&lng=(.*?)&", href).groups()
         name = bank.xpath(".//h3/text()").extract_first()
-        ref = name + "-" + "-".join(address[0].split(" "))
+        ref = "-".join(name.split(" ")) + "-" + "-".join(address[0].split(" "))
```

Comment:
All the Places generates IDs based on a hash of the `ref` and the spider name, so we prefer to keep the `ref` consistent where possible. Unless there's a specific reason you made this change, can we roll this particular line back to keep the name consistent?

```suggestion
        ref = name + "-" + "-".join(address[0].split(" "))
```

Reply:
Fixed